civihr_default_theme/assets/sass/civihr_default/_custom.scss

```diff
@@ -1045,3 +1045,19 @@ form {
 .jcarousel-container [disabled] {
     display: none;
 }
+
+/* Styling the exposed filters on reports page. */
+@media (min-width: 768px) { 
```

Comment:
@brunoqueiros Coz the style it was taking on enabling @import "bootstrap/input-groups, and @import "bootstrap/forms"; was also in this media query. Maybe we need full width on mobile devices.